src/components/tabs/tabs.scss

```diff
@@ -0,0 +1,49 @@
+@use "../../settings/constants";
+
+.mykn-tabs {
+  &__buttons {
```

Comment:
I think the name "buttons" is a bit vague, can we rename this to head or something more descriptive?